docs/docs/behaviors/key-tap.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Key Tap Behavior
+sidebar_label: Key Tap
+---
+
+## Summary
+
+The key tap behavior is a combination of key press and release on just pressing the key.
+
+All keycodes including modifiers can be used the same way as with the key press behavior (so standard keycodes are sent for press/release).
+
+It is usefull if you want to make a layout that has the same keycode with and without a modifier close together, e.g. `[` and `{`.
+
+Since `LBRC` is basically just `LSHIFT(LBKT)` it can happen, that you still hold `{` while using `[` which leads to a missed keycode since the base-key is still pressed.
+
+By immediately releasing the key with key tap this is circumvented.
```

Comment:
This is a great tip that I think would be nice wrapped up inside a "tip" box, with a link to the GitHub thread tracking the issue. 

I personally find the simplified `+=` example from the GitHub thread is a little clearer to comprehend than the more detailed `{[` brackets example, but it's also then consistent when clicking through to GitHub.

```suggestion
:::tip

Key tap is useful if you experience issues when rolling keys that share a base keycode.

For example, a common programming bigram is `+=`. Behind the scenes, `+` is produced with `SHIFT` and `=`. So typing the keys `+` and `=` too quickly can sometimes result in one of them being ignored since the base-key is still pressed. However, by immediately releasing the key with key tap this issue is circumvented.

See [zmkfirmware/zmk #1076](https://github.com/zmkfirmware/zmk/issues/1076) for a thread tracking this.

:::
```

The above suggestion looks like this on my local environment:

<img width="964" alt="image" src="https://user-images.githubusercontent.com/60922914/170971166-cdb2f381-ba5e-43b9-94e6-e502ed8eec8d.png">